src/consensus/pbft/pbftconfig.h

```diff
@@ -64,23 +64,29 @@ namespace pbft
           pbft::Request request;
           request.deserialise({inb->contents, inb->contents + inb->size});
 
-          LOG_DEBUG_FMT("PBFT exec_command() for frontend {}", request.actor);
+          const enclave::SessionContext session(
+            enclave::InvalidSessionId, request.caller_id, request.caller_cert);
+          auto ctx = enclave::make_rpc_context(
+            session, request.raw, {req_start, req_start + req_size});
 
-          auto handler = this->rpc_map->find(ccf::ActorsType(request.actor));
+          const auto actor_opt = http::extract_actor(*ctx);
```

Comment:
This pattern is repeated in several places. Given an `RpcContext` constructed from a raw byte representation, we want to find the frontend to pass it to. I think this should be reduced to `rpc_map->process(ctx)`, and no-one else needs to care about actors.